### ISSUE TITLE
Data loss

### ISSUE THREAD

**Issue by j-anstee** [CSSEGISandData/COVID-19#653]
California has been removed from US counts today

**Comment by kamronbatman** [comment]
Yeah I just noticed the numbers went down in the US and tried to drill into CA but it said we have no cases! haha

**Comment by eilifm** [comment]
+1 Here. Data for the US is now unreliable.

**Comment by raysalem** [comment]
so is the data on portal correct and i it derived from this data set:

https://www.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6

**Comment by heavenly-star** [comment]
CDC has been waiting on sign-offs by governors before adding to the CDC website which is what this dashboard is depending on. For example, the CDC shows the breakdown by state if you scroll down about midway to view orange map at https://www.cdc.gov/coronavirus/2019-ncov/cases-in-us.html#investigation. You will notice that CDC shows Florida has 33 cases. However, according to the Florida governor's website is 56 positives at http://www.floridahealth.gov/diseases-and-conditions/COVID-19/index.html.

**Comment by Pamblam** [comment]
missing data for southeastern us too.

**Comment by NoShotz** [comment]
Canada is missing in the countries list, either that or i'm blind.

**Comment by ragzilla** [comment]
Likely related to #650
